Sources/App/Views/PackageController/GetRoute.Model+ext.swift

```diff
@@ -298,6 +298,17 @@ extension API.PackageController.GetRoute.Model {
         )
     }
 
+    func targetTypeListItem(_ type: Target.TargetType) -> Node<HTML.ListContext> {
+        guard let targets = targets
+        else { print("hello"); return .empty }
```

Comment:
`print("hello")` eh? 😅

```suggestion
        else { return .empty }
```

Reply:
What do you mean my debugging techniques are primitive and ridiculous? That's a scandalous accusation!

😬